api/v1beta1/redis_types.go

```diff
@@ -26,21 +26,20 @@ import (
 
 // RedisSpec defines the desired state of Redis
 type RedisSpec struct {
-	KubernetesConfig  KubernetesConfig             `json:"kubernetesConfig"`
-	RedisExporter     *RedisExporter               `json:"redisExporter,omitempty"`
-	RedisConfig       map[string]string            `json:"redisConfig"`
-	Resources         *corev1.ResourceRequirements `json:"resources,omitempty"`
-	Storage           *Storage                     `json:"storage,omitempty"`
-	NodeSelector      map[string]string            `json:"nodeSelector,omitempty"`
-	SecurityContext   *corev1.PodSecurityContext   `json:"securityContext,omitempty"`
-	PriorityClassName string                       `json:"priorityClassName,omitempty"`
-	Affinity          *corev1.Affinity             `json:"affinity,omitempty"`
-	Tolerations       *[]corev1.Toleration         `json:"tolerations,omitempty"`
+	KubernetesConfig  KubernetesConfig           `json:"kubernetesConfig"`
+	RedisExporter     *RedisExporter             `json:"redisExporter,omitempty"`
+	RedisConfig       map[string]string          `json:"redisConfig"`
+	Storage           *Storage                   `json:"storage,omitempty"`
+	NodeSelector      map[string]string          `json:"nodeSelector,omitempty"`
+	SecurityContext   *corev1.PodSecurityContext `json:"securityContext,omitempty"`
+	PriorityClassName string                     `json:"priorityClassName,omitempty"`
+	Affinity          *corev1.Affinity           `json:"affinity,omitempty"`
+	Tolerations       *[]corev1.Toleration       `json:"tolerations,omitempty"`
 }
 
 // RedisStatus defines the observed state of Redis
 type RedisStatus struct {
-	Redis RedisSpec `json:"redis,omitempty"`
+	Redis RedisSpec `json:"redis"`
```

Comment:
This should be a required field, given that its critical..

Reply:
Agreed